GitHub actions

markrtuttle opened the pull request:
This fixes the GitHub Actions building CBMC packages.
* The Debian patch 64bit_regression_tests.patch failed to apply due to recent changes in regression/cbmc/xml-trace/test.desc
* The workflow was changed to build only the packages that install into the operating system's equivalent of /usr/local/bin (the installation into /usr/local/cbmc-latest/bin turns out to be kind of useless).

By the way, the workflow Build and Test Xen also appears to be broken in develop.

Commit:
Modify GitHub Actions patch of Debian 64bit_regression_tests.patch file.

Commit:
Modify GitHub Actions to build only /usr/local/bin installers.

The GitHub Actions builds one set of installers that installs into the
operating system's equivalent of /usr/local/bin, and a second set of
installers that installs into the equivalent of
/usr/local/cbmc-latest/bin.  This patch removes the second set of
installers and leaves only those installing into /usr/local/bin.

kroening commented on .github/workflows/packages.yaml:
```diff
@@ -105,43 +105,43 @@ jobs:
-
-  Ubuntu18_Stable:
+#  Ubuntu16_Latest:
+#    name: Ubuntu 16 cbmc-latest package
```
Simply get rid of the commented lines?

kroening commented on .github/workflows/packages.yaml:
```diff
@@ -242,19 +242,19 @@ jobs:
           make -f ${SCRIPT_DIR}/Makefile
           mv cbmc.tar.gz ${PACKAGE_STABLE_NAME}
-          mv cbmc-latest.tar.gz ${PACKAGE_LATEST_NAME}
+#          mv cbmc-latest.tar.gz ${PACKAGE_LATEST_NAME}
```
Also remove PACKAGE_LATEST_NAME environment variable above.

NlightNFotis reviewed (commented):
I second @kroening 's comments on removing the (now dead) build instructions, otherwise it's good to go.

Please implement this and I will approve and merge.

Commit:
Clean up packages.yaml.

Delete unneeded portions of packages.yaml (don't just comment them
out), and delete all unneeded environment variable definitions.

NlightNFotis reviewed (approved):
Thanks for implementing the changes asked.

This is now good to go, and thus I will merge it.
